Config.lua

```diff
@@ -537,7 +537,9 @@ end
 function addon:InitConfigOptions()
 	profile = addon.db.profile
 	mainOptions.args.profiles = self.ACDBO:GetOptionsTable(self.db)
-	addon.LDS:EnhanceOptions(mainOptions.args.profiles, self.db)
+	if(not self:IsClassic() and not self:IsBCC()) then
+		addon.LDS:EnhanceOptions(mainOptions.args.profiles, self.db)
+	end
 	addon.AC:RegisterOptionsTable("VocalRaidAssistantConfig", mainOptions)
```

Comment:
Check WoW Version during init and use variable to load according config

Reply:
I'm not sure exactly what you mean. Could you give an example? Do you mean a single variable determining the version or three separate isRetail, isClassic, isTBC variables, and how would that help?